README.md

```diff
@@ -0,0 +1,41 @@
+# Overview
+Simulator is a passive th2 component with parameterised rules which is implemented simulate logic.
+Script or other components can create, remove and get information about running rules in simulator via gRPC. Simulator can interact with conn (Connects).
```

Comment:
Let's replace information related to the sim (core) feature to link to the th2-sim project.